### ISSUE TITLE
Show block windows for *this week* in the console

### ISSUE THREAD

**Issue by bratseth** [vespa-engine/vespa#20708]
We expose the hours and days block window in application API which is used in console, f.ex.:
![image](https://user-images.githubusercontent.com/3785505/148580627-c329203f-ad6d-45b4-8f06-6afd8c639798.png)

https://github.com/vespa-engine/vespa/blob/a3bc6f9b6af95b97a2b55147a586ca19d1d98938/controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java#L1162
I think we should change this to be based on _this_ week (as opposed to what it is now, which is the days/hours blocked without date range).

_Originally posted by @freva in https://github.com/vespa-engine/vespa/pull/20699#pullrequestreview-846809611_

**Comment by mpolden** [comment]
I don't think it makes sense to show anything beyond the ETA for when the application/platform windows opens up next.

In any case I think this requires a Console hacker.

**Comment by baldersheim** [comment]
soon has timed out, still relevant ?
